### DISCUSSION TITLE
docs/v1.11.x/start/sealos-install/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#2351] [Announcements]
# docs/v1.11.x/start/sealos-install/

https://kubeovn.github.io/docs/v1.11.x/start/sealos-install/

**Comment by wangsqly0407** [comment]
按照文档部署，报错如下：
message: 'render env to rootfs failed: failed to create template: /var/lib/containers/storage/overlay/6020074e76c2853e968c787c51e33e981866bb29d86db5c9d242a1264bb3669c/merged/etc/image-cri-shim.yaml.tmpl template: image-cri-shim.yaml.tmpl:21: function "semverCompare" not defined' reason: ApplyClusterError

解决方案：升级到 sealos 4.1.4 版本即可

**Reply by oilbeater** [reply]
已更改文档，感谢发现

**Comment by shen-buding** [comment]
[Step 4/6] Delete pod that not in host network mode
pod "coredns-6d4b75cb6d-r7dtf" deleted
pod "coredns-6d4b75cb6d-rqfrj" deleted
pod "kube-ovn-pinger-jncp2" deleted
daemon set "kube-ovn-pinger" successfully rolled out
Waiting for deployment "coredns" rollout to finish: 0 of 2 updated replicas are available...
error: timed out waiting for the condition
2023-03-21T23:24:39 error Applied to cluster error: exit status 1



我这边coredns遇到了点问题
